### DISCUSSION TITLE
disk order

### DISCUSSION THREAD

**Discussion by oshmoz** [aristocratos/btop#320] [Q&A]
Hello,
Really love btop. Just a quick question, how do I order the disks list? The current order display my least used disks at the top.
Thanks.

**Comment by einsern** [comment]
efi and boot really aren't the disks I care about in a activity monitor.

**Reply by zen0bit** [reply]
disks_filter = "exclude=/boot /boot/efi"

**Reply by oshmoz** [reply]
But can you change the order of the displayed disks?

**Comment by zen0bit** [comment]
how to get rid of efi drive?

**Reply by zen0bit** [reply]
solved
